.github/workflows/cli_setup.yml

```
@@ -0,0 +1,76 @@
+name: CLI Setup
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    paths:
+    - 'qmk_cli/**'
+    - 'milc.py'
+    - 'qmk'
+    - 'requirements.txt'
+    - '.github/workflows/setup_and_doctor.yml'
+    - 'setup.cfg'
+    - 'setup.py'
+
+jobs:
+  test_cli_nix:
```

Nix is also a package manager which makes this name a little confusing.

```suggestion
  test_cli_linux_macos:
```